.github/workflows/continuous_integration.yml

```
@@ -0,0 +1,58 @@
+---
+name: Continuous Integration
+on:
+  push:
+    branches:
+      - main
+      - candlepin-4.1-HOTFIX
+      - candlepin-4.2-HOTFIX
+
+env:
+  JAVA_DISTRIBUTION: 'temurin'
+
+jobs:
+  internationlization:
+    name: internationlization
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:38
+
+    steps:
+      - name: Mask secrets
+        shell: bash
+        run: |
+          echo "::add-mask::${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Install dependencies
+        shell: bash
+        run: dnf --setopt install_weak_deps=False install -y gettext jss git
+
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Set up Java 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: ${{ env.JAVA_DISTRIBUTION }}
+          java-version: 11
+
+      - name: Set up Java 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: ${{ env.JAVA_DISTRIBUTION }}
+          java-version: 17
+
+      - name: Add safe directory
+        shell: bash
+        run: git config --system --add safe.directory /__w/candlepin/candlepin
+
+      - name: Run internationlization
```

```suggestion
      - name: Run i18n
```
(there was a typo in the name, and IMHO it can be shortened)

---

Updated!